packages/e2e-tests/specs/editor/blocks/navigation.test.js

```diff
@@ -1463,22 +1703,22 @@ Expected mock function not to be called but it was called with: ["POST", "http:/
 			// Insert new block and wait for the insert to complete.
 			await insertBlock( 'Navigation' );
 			await waitForBlock( 'Navigation' );
-			await page.waitForXPath( START_EMPTY_XPATH );
 
-			// Change menus via the select menu toolbar button.
-			const selectMenuDropdown = await page.waitForSelector(
-				'[aria-label="Select Menu"]'
+			const navigationSelector = await page.waitForXPath(
+				"//button[text()='Select Menu']"
 			);
-			await selectMenuDropdown.click();
-			const exampleNavigationOption = await page.waitForXPath(
-				'//span[contains(text(), "Second Example Navigation")]'
+			await navigationSelector.click();
+
+			const theOption = await page.waitForXPath(
+				"//button[@aria-checked='false'][contains(., 'First navigation')]"
 			);
-			await exampleNavigationOption.click();
+			theOption.click();
```

Comment:
@draganescu There's another `await` here. Also noticed this test seems to be skipped now?